toolchains/gcc_compatibles.cmake

```diff
@@ -22,9 +23,16 @@ set( TNUN_compiler_exceptions_off      -fno-exceptions                       )
 set( TNUN_compiler_optimize_for_speed  -O3                                   )
 set( TNUN_compiler_optimize_for_size   -Os                                   )
 set( TNUN_compiler_report_optimization -ftree-vectorizer-verbose=6           )
-set( TNUN_compiler_release_flags       -DNDEBUG -fomit-frame-pointer -ffunction-sections -fdata-sections -fmerge-all-constants -fno-stack-protector )
+set( TNUN_compiler_release_flags       -DNDEBUG -fomit-frame-pointer -ffunction-sections -fmerge-all-constants -fno-stack-protector )
+if( ${CMAKE_SYSTEM_NAME} MATCHES "Linux" )
```

Comment:
The original idea was to also have this flag on android, but since it was added in android toolchain, it can be removed from here.